assets/dev/js/editor/document/elements/commands-internal/reset-settings.js

```diff
@@ -0,0 +1,13 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export class ResetSettings extends CommandBase {
+	validateArgs( args ) {
+		this.requireContainer( args );
+	}
+
+	apply( args ) {
+		$e.run( 'document/elements/reset-settings', { ...args, options: { useHistory: false } } );
```

Comment:
Internal command that runs public command, sound non-logical, which issue did you face?

Reply:
As discussed, I removed the internal command and used the regular one.